src/components/relation-list/index.tsx

```diff
@@ -204,9 +204,15 @@ const RelationsList: FC<Props> = ({
           <SC.RelationLinks>
             {publicServices.map(({ uri, title, id }) =>
               uri && id && title ? (
-                <Link as={RouterLink} to={`${PATHNAME_PUBLIC_SERVICES}/${id}`}>
-                  {translate(title ?? uri)}
-                </Link>
+                <span>
```

Comment:
Kan være `<>` i stedet for `<span>`

Reply:
Teksten skal være inline med linken, er det ikke cleanest å gjøre dette med `span`?

Reply:
det spørs på om fragment gir samme effekt som span. hvis ja, fordelen er mindre karakterer, hvis ikke, da kan du ignorere meg